.github/workflows/test-demo-native-marketplace.yml

```diff
@@ -10,7 +10,6 @@ on:
       - "20[0-9][0-9][0-1][0-9][0-3][0-9]*"
   schedule:
     - cron: "0 0 * * 1"
-  pull_request:
```

Comment:
why this change?

Reply:
The thought was that we can just run marketplace test manually for now.

Reply:
reverted 4e9f7428f7b2a5d7190bcd64b73ce301f7c77496